autocomplete/complete.go

```diff
@@ -94,64 +104,188 @@ func getPotentialTarget(prefix string) ([]string, error) {
 	return potentials, nil
 }
 
-func getPotentialPaths(prefix string) ([]string, error) {
-	if prefix == "." {
-		return []string{"./", "../"}, nil
+func getUser(username string) (*user.User, error) {
+	username = strings.Replace(username, "~", "", 1)
+	return user.Lookup(username)
+}
+
+func getUsers() ([]string, error) {
+	u, err := user.Current()
+	if err != nil {
+		return nil, err
+	}
+
+	lPath := strings.Split(u.HomeDir, "/")
```

Comment:
Instead of splitting this, would it be possible to use `filepath.Dir(u.HomeDir)` from the `path/filepath` package instead?

Reply:
done ;)